src/css/media-queries.scss

```diff
@@ -0,0 +1,103 @@
+$break-mobile: 420px;
+$break-tablet: 421px;
+$break-largeSize: 775px;
+
+@media only screen and (max-width: $break-mobile) {
+  
+  .header {
+    height: 250px;
+    display: flex;
+    flex-direction: column;
+    h1 {
+      font-size: 2em;
+    }
+    p .header-welcome {
+      font-size: .6em;
+    }
+    .dropdown {
+      position: unset;
+    }
+    .header-date {
+      font-size: .7em;
+    }
+    .switch__dropdown {
+      display: flex;
+      margin-bottom: 10px;
+    }
+    .switch {
+      margin-right: 35px;
+    }
+  }
+
+}
+
+@media screen and (min-width: $break-tablet) {
+
+  .header {
+    height: 250px;
+    display: flex;
+    flex-direction: column;
+    h1 {
+      font-size: 2.5em;
+    }
+    p .header-welcome {
+      font-size: .8em;
+    }
+    .dropdown {
+      position: unset;
+    }
+    .header-date {
+      font-size: 1em;
+    }
+    .switch__dropdown {
+      display: flex;
+      margin-bottom: 20px;
+      justify-content: space-between;
+    }
+    .switch {
+      margin-right: 135px;
+    }
+    ul {
+      width: 50%;
+    }
+  }
+}
+
+@media screen and (min-width: $break-largeSize){
+
+  .header {
+    // background-color: lavender;
```

Comment:
Got an unused commented out section. If you can remove this, I will go ahead with the approval